tests/kubernetes/graph/checks/test_yaml_policies.py

```diff
@@ -28,6 +30,8 @@ def __init__(self, args):
             db_connector = NetworkxConnector()
```

Comment:
Since this code snippet of fetching db_connector based on graph framework appears in many tests, is it possible to extract this to a common place ? It will also make it easier when we remove igraph.

Reply:
yes, it's possible. 
added utils for db_connector